scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailStatus.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.scheduler.core;
+
+import org.ow2.proactive.scheduler.common.job.JobInfo;
+
+
+/**
+ * The JobEmailStatus defines the status of the job for email notification.
+ * The status of the job could be either aborted (due to an error policy, manual kill but the user,
+ * or due to resource failure). or it could finish with errors (when at least one task fails),
+ * or finally it could finish without issues when the two previous tasks are false.
+ *
+ * @author ActiveEon Team
+ * @since 2020-01-07
+ */
+public class JobEmailStatus {
```

Comment:
I guess this class can be used for any other case than Email notification (I'm thinking, at list, to other types of notifications)

Reply:
no, only emaill notification